extension/src/standalone_pages/InstructorDashboardRenderVideoInfo.js

```diff
@@ -39,4 +72,145 @@ function renderVideoAccordian(document, videoObjects) {
     document.getElementById("nav-videos").appendChild(accordianDiv);
 }
 
-export { renderVideoAccordian };
+function renderActiveFeedback(video, response) {
+    let color= "#5959e6";
+    let activeChart = document.getElementById(`activeFeedback_${video.videoID}`).getContext("2d");
+    response.active_questions.sort((a, b) => {
+        return a.timestamp - b.timestamp;
+    });
+    let labels = response.active_questions.flatMap((question) => {
+        if (question.timestamp !== null) {
+            return [question.timestamp];
+        } else {
+            return [];
+        }
+    });
+    let amounts = response.active_questions.flatMap(question => {
+        if (question.count !== null) {
+            return [question.count];
+        } else {
+            return [];
+        }
+    });
+    let questionsMap = new Map();
+    labels.forEach((timestamp) => {
+        questionsMap[timestamp] = response.active_questions_text.filter(question => question.timestamp === timestamp);
+    });
+    let chart = new Chart(activeChart, {
+        type: "bar",
+        data: {
+            labels: labels,
+            datasets: [{
+                label: "Active Feedback",
+                borderColor: color,
+                pointBackgroundColor: color,
+                pointBorderColor: color,
+                pointHoverBackgroundColor:color,
+                pointHoverBorderColor: color,
+                data: amounts,
+            }],
+        },
+        options: {
+            scales: {
```

Comment:
Let's fixate the range of the x-axis. The end x axis is given as part of the video info

Reply:
Unfortunately, I don't think there's a way to do this.

I don' think it's a problem since we shouldn't have passive or active feedback that extends outside the video_duration.

If so, I'd prefer to come back and fix the bug after implementing other more urgent features.

Reply:
Have you try this setting?

https://www.chartjs.org/docs/latest/axes/cartesian/linear.html#axis-range-settings

We can move forward without this UI fix if this does not work

Reply:
Max and Min settings do not seem to work on X labels.

Added manual filtering to compensate.